src/task6.py

```diff
@@ -0,0 +1,8 @@
+import re
```

Comment:
here you have 0 lines between import and code (and 1 in other tasks). Your code should be consistent